sass/components/_funding.scss

```diff
@@ -0,0 +1,3 @@
+#project-funding .ecl-content-item__meta {
```

Comment:
Let's rename this file to _content_project.scss to use for the whole Project page.

Reply:
Changed to _content_project.scss